src/heretic/model.py

```diff
@@ -48,6 +49,10 @@ def __init__(self, settings: Settings):
             self.tokenizer.padding_side = "left"
 
         self.model = None
+        self.trusted_models = {settings.model: settings.trust_remote_code}
+
+        if self.settings.evaluate_model is not None:
+            self.trusted_models[settings.model] = settings.trust_remote_code
```

Comment:
I don't understand this line. Doesn't it do exactly the same thing as line 52?

Reply:
Whoopsie, that should've been the `evaluate_model` obviously